src/script/filterByText.js

```diff
@@ -10,21 +10,23 @@ export class FilterByText {
         const searchInput = document.querySelector('.input-search-keyword');
         const preSearchString = searchInput.value;
         const searchString = preSearchString.toLowerCase();
```

Comment:
If you change this line to:
const searchString = searchInput.value.toLowerCase();
you can remove line 11 and change all preSearchString to just searchString. It gets less confusing that way :D

Reply:
Thanks, will change that.